recipes/watchdog-wms/build.sh

```diff
@@ -19,12 +19,16 @@ mv "${OUT}/workflowDesigner.sh" "${OUT}/watchdog-gui.sh"
 ln -s "${OUT}/watchdog-cmd.sh" "${PREFIX}/bin/watchdog-cmd"
 chmod 0755 "${PREFIX}/bin/watchdog-cmd"
 
+# get the modules
+/bin/bash "${OUT}/modules/downloadCommunityModules.sh" --auto
+
 # test, if javafx is working
 cd "${RECIPE_DIR}"
 RET=$(java "TestJavaFx")
 if [ "$RET" == "INSTALLED" ]; then
 	ln -s "${OUT}/watchdog-gui.sh" "${PREFIX}/bin/watchdog-gui"
 	chmod 0755 "${PREFIX}/bin/watchdog-gui"
+	echo "Found Javafx."
```

Comment:
I won't pretend to be a java expert, but this is only telling you what was in the build environment. If openjdk is going to support Javafx going forward then that's fine, otherwise I think this would be better as a post-install script.

Reply:
You're right. I will change that.

Reply:
I removed that test completely becuase I have a javafx detection in watchdog-gui.sh anyway.